External/FEXCore/Source/Interface/Core/OpcodeDispatcher.h

```diff
@@ -513,6 +513,26 @@ friend class FEXCore::IR::PassManager;
 
   bool Multiblock{};
   uint64_t Entry;
+
+  IRPair<IROp_StoreMemTSO> _StoreMemTSO(FEXCore::IR::RegisterClassType Class, uint8_t Size, OrderedNode *ssa0, OrderedNode *ssa1, uint8_t Align = 1) {
```

Comment:
Make these be helpers that don't follow the same naming convention as the IREmitter functions.
It's confusing that an IR "emission" of _StoreMemTSO doesn't always do a _StoreMemTSO Op.
Also should just make the helper return OrderedNode*, doesn't feel good converting the IRPair to TSO, if one happens to change definition then this would immediately be a broken cast.

Reply:
`_StoreMemAutoTSO`? Maybe also get rid of the `_`?

Yeah, agreed on the return, though we cast the structs to each other all over the code. Will change it

Reply:
Renamed to `_StoreMemAutoTSO`